subsys/logging/CMakeLists.txt

```diff
@@ -13,3 +13,8 @@ zephyr_sources_ifdef(
   CONFIG_LOG_BACKEND_UART
   log_backend_uart.c
   )
+
+zephyr_sources_ifdef(
+  CONFIG_LOG_BACKEND_NATIVE
```

Comment:
native of which country? ;) to user who do not care about native posix shortcut 'native' does not tell anything thus imo this should rather be log_backend_native_posix.